packages/icon/src/IconBase.ts

```diff
@@ -36,6 +36,7 @@ export class IconBase extends SpectrumElement {
         if (changes.has('label')) {
             if (this.label) {
                 this.removeAttribute('aria-hidden');
+                this.setAttribute('aria-label', this.label);
```

Comment:
Per the [readme](https://github.com/adobe/spectrum-web-components/tree/main/packages/icon#accessibility) for icon:

aria-hidden is set to true by default for Icons. The label attribute suppresses this and adds the label text as the aria-label of the icon.

This was not implement so fixed it in this PR

Reply:
I wonder why we don't use `aria-label` from the component tag instead of introducing a new `label` property? I removed the custom `label` property and use the `aria-label` that can be specified at the component level

Reply:
At the API level we _generally_ use `label` because sometimes that element must be labeled at the host (which would be the same as using `aria-label`, but other times that label needs to be applied to something _within_ the host, and forwarding `aria-label` is a tricky accessibility/timing cup game of dispair. Having a single entry point across as many elements as possible intends that it is less likely that a consumer makes the mistake of using the wrong one when switching back and forth between them over the course of developing a larger application.